test/traceback-styles/results.native.json

```diff
@@ -0,0 +1,10 @@
+{
+  "status": "fail",
+  "tests": [
+    {
+      "name": "TracebackStylesTest.test_hello",
+      "status": "fail",
+      "message": "Traceback (most recent call last):\n  File \"/home/michael/Documents/dev/python-test-runner/test/traceback-styles/traceback_styles_test.py\", line 9, in test_hello\n    self.assertEqual(hello(), \"Goodbye\")\n  File \"/usr/lib/python3.8/unittest/case.py\", line 912, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/usr/lib/python3.8/unittest/case.py\", line 1292, in assertMultiLineEqual\n    self.fail(self._formatMessage(msg, standardMsg))\n  File \"/usr/lib/python3.8/unittest/case.py\", line 753, in fail\n    raise self.failureException(msg)\nAssertionError: 'Hello, World!' != 'Goodbye'\n- Hello, World!\n+ Goodbye"
```

Comment:
Is the path here going to be a problem for the unit tests?

Reply:
Definitely, working on fixing that.